ajax/libs/jeditable.js/package.json

```diff
@@ -1,17 +1,37 @@
 {
   "name": "jeditable.js",
-  "filename": "jeditable.min.js",
-  "version": "1.7.3",
-  "author": "Mika Tuupola",
-  "description": ". With few lines of JavaScript code I allow you to click and edit the content of different html elements. I am based on Dylan Verheul’s editable.",
-  "homepage": "http://www.appelsiini.net/projects/jeditable",
+  "filename": "jquery.jeditable.min.js",
+  "version": "1.8.0",
+  "author": "Mika Tuupola <tuupola@appelsiini.net>",
+  "description": "jQuery edit in place plugin. Extendable via plugin architecture. Plugins for plugin. Really.",
+  "homepage": "https://www.appelsiini.net/projects/jeditable",
   "keywords": [
     "editable",
-    "jquery"
+    "jquery",
+    "edit",
+    "in",
+    "place",
+    "jeditable",
+    "plugin"
   ],
   "repository": {
     "type": "git",
     "url": "https://github.com/tuupola/jquery_jeditable.git"
   },
+  "npmName": "jquery-jeditable",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
```

Comment:
This is my mistake in first review.
It's better to use git auto-update, because there is more versions from GitHub repo (3 versions) then npm package(1 versions).
So use git auto-update config as the following is better.
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/NicolasCARPi/jquery_jeditable.git",
    "fileMap": [
      {
        "basePath": "dist",
        "files": [
          "**/*"
        ]
      },
      {
        "basePath": "src",
        "files": [
          "jquery.jeditable*"
        ]
      }
    ]
  }
```

Would you modify it in second commit?

If this would bother you, please let me know, I would help this PR better to be merged.

Many thanks.

Reply:
https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md#enabling-npmrecommended-or-git-auto-update
Mentions that you guys preferred npm, which is why I used that over git. 

I see the new repo owner now has a new site for jeditable, https://jeditable.elabftw.net/
So looks like the url prob should be updated now to reflect that as well.

Reply:
@thezoggy Actually, we hope CDNJS can host more versions, because this could provide more choose to developers.
That's why choose git auto-update config is better than npm auto-update config.

What do you think?

Also, would you please help improve this PR?
If this would bother you, please let me know, I can help this PR.

Many thanks.

Reply:
I can swap it out to use git, I originally had it doing it but switched per the docs to use npm.